_includes/docs-info.html

```diff
@@ -42,17 +42,19 @@ <h6>For Users</h6>
         <a name="tutorials"></a>
         <h5>Tutorials for the finer steps</h5>
             <div class="row docs-details">
-                <a href="https://bit.ly/2Fyn97v">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated July 21st 2017.
+                <a href="https://bit.ly/3q73CTo">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2WsFiun">Study-specific shape analysis (GROUPS)</a> Last updated July 16th 2018.
+                <a href="https://bit.ly/3q6v1oo">Study-specific shape analysis (GROUPS)</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/2uVYche">Shape Regression</a> Last updated February 5th 2019.
+                <a href="https://bit.ly/3KKLWGO">Shape Regression</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2HYbHVA">Shape Variation Analyzer</a> Last updated December 9th 2019.
+                <a href="https://bit.ly/3cHG704">Shape Variation Analyzer</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/30MiJ6v">Shape analysis via skeletal models</a> Last updated July 1st 2019.
+                <a href="http://bit.ly/30MiJ6v">Shape analysis via skeletal models</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/319KVBN">Covariate Significance Testing</a> Last updated July 28th 2020.
+                <a href="http://bit.ly/319KVBN">Covariate Significance Testing</a> Last updated September 2022.
```

Comment:
https://docs.google.com/presentation/d/1AsdtNDKuX3a14U1gm5gUJfVE-rOio1znPmg1fSmN83Y/present?slide=id.p1

This has the old Kitware logo.

Reply:
https://docs.google.com/presentation/d/1AsdtNDKuX3a14U1gm5gUJfVE-rOio1znPmg1fSmN83Y/present?slide=id.p6
https://docs.google.com/presentation/d/1AsdtNDKuX3a14U1gm5gUJfVE-rOio1znPmg1fSmN83Y/present?slide=id.g8d2445b938_1_3

These are the old website images.

Reply:
https://docs.google.com/presentation/d/1AsdtNDKuX3a14U1gm5gUJfVE-rOio1znPmg1fSmN83Y/present?slide=id.p19

I cannot click on the link for the MFSDA github repo.

Reply:
Kitware logo has been fixed.
Old website images have been fixed.
MFSDA link has been fixed.